phpBB/includes/acp/acp_groups.php

```diff
@@ -293,7 +296,19 @@ function main($id, $mode)
 				// Add user/s to group
 				if ($error = group_user_add($group_id, false, $name_ary, $group_name, $default, $leader, 0, $group_row))
 				{
-					trigger_error($user->lang[$error] . adm_back_link($this->u_action . '&amp;action=list&amp;g=' . $group_id), E_USER_WARNING);
+					$display_message = $language->lang($error);
+
+					if ($error == 'GROUP_USERS_INVALID')
+					{
+						// Find which users don't exist
+						$actual_name_ary = $name_ary;
+						$actual_user_id_ary = false;
```

Comment:
Why are you setting this to a boolean value? user_get_id_name expects an array.

Reply:
@marc1706 There's lots of places in the code where we already treat it as a boolean. Like when [`group_user_attributes()`](https://github.com/phpbb/phpbb/search?q=group_user_attributes&unscoped_q=group_user_attributes), [`group_user_del()`](https://github.com/phpbb/phpbb/search?q=group_user_del&unscoped_q=group_user_del) or [`group_user_add()`](https://github.com/phpbb/phpbb/search?q=group_user_add&unscoped_q=group_user_add) get called. 

Would you like me to change mine to `$actual_user_id_ary = array();` or change the `@param` type to mixed in the function seeing we already use booleans in quite a few areas?